server/src/main/kotlin/fi/oph/kitu/yki/YkiSuoritusEntity.kt

```diff
@@ -0,0 +1,31 @@
+package fi.oph.kitu.yki
+
+import fi.oph.kitu.yki.enums.Tutkintokieli
+import fi.oph.kitu.yki.enums.Tutkintotaso
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Table
+import java.util.Date
+
+@Table(name = "yki_suoritus")
+class YkiSuoritusEntity(
+    @Id
+    val id: Number?,
+    val suorittajanOppijanumero: String,
```

Comment:
Jätin koton puolella nämä snake_caseksi koska ne tulevat koealustalta silleen ja tallennetaan kantaan silleen, mutta ehkä ne kannattaisi nimetä näin idiomaattisuuden vuoksi. 🤔

Reply:
Ajattelin, että camelCase on koodauskäytännöiltään parempi kuin snake_case. Spring framework:han osaa mapata noi vastaaviin snake_case kolumneihin tietokannassa.